src/content/guides/getting-started.md

```diff
@@ -204,7 +205,7 @@ Open `index.html` in your browser and, if everything went right, you should see
 
 The [`import`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import) and [`export`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export) statements have been standardized in [ES2015](https://babeljs.io/learn-es2015/). Although they are not supported in most browsers yet, webpack does support them out of the box.
 
-Behind the scenes, webpack actually "transpiles" the code so that older browsers can also run it. If you inspect `dist/bundle.js`, you might be able to see how webpack does this, it's quite ingenious! Besides `import` and `export`, webpack supports various other module syntaxes as well, see [Module API](/api/module-methods) for more information.
+Behind the scenes, webpack actually "transpiles" the code so that older browsers can also run it. If you inspect `dist/main.js`, you might be able to see how webpack does this, it's quite ingenious! Besides `import` and `export`, webpack supports various other module syntaxes as well, see [Module API](/api/module-methods) for more information.
```

Comment:
I think that transpile is not the right term and will actually confuse not experienced users. It's true that webpack does some small changes on the require statements, but most of all it just adss an overhead to crawl thourgh the modules.

Do you think we could rephrase this?

Reply:
the sentence itself is catchy so maybe just change `"transiples"` into `transforms` ?

Reply:
Maybe in another PR? We could use this PR to finally settle that filename change.

Reply:
okay 👍